src/rewriters/Rewriter.h

```diff
@@ -78,6 +79,7 @@ class DefaultRewriterConfig {
 public:
     virtual bool previsit(PTRef term) { return true; } // should continue visiting
     virtual PTRef rewrite(PTRef term) { return term; } // don't do anything
+    virtual bool treatIteAsVar() const { return true; }
```

Comment:
I think this should not be added here. We don't need this functionality anymore.

Reply:
Removed.